### DISCUSSION TITLE
Train Pretrained NN

### DISCUSSION THREAD

**Discussion by joschikbrunow** [BrainJS/brain.js#710] [Q&A] [answered]
Is it possible to train a pretrained neural network? For example: You import an JSON-NN, detecting letters. Then you want to teach the NN another letter, for instance the Germann letter "ß", without repeat learning the old letters.

**Comment by robertleeplummerjr** [comment, marked as answer]
At this time this is not supported.

**Reply by robertleeplummerjr** [reply]
To be clear, I'm not even sure there are _any_ libraries that support this directly.  The way around it could be to start a network with some extra inputs, and never activate them, then you could use them, but sometimes that can have unintended consequences.
